Add new rllib agent

oliverwang15 opened the pull request:
* The new agent adapts to ray >= 1.13.0
* Changes in `DRL_prediction` function. The function returns a `rllib_agent` which can output actions according to the saved model and input observations so that we can customize our backtest process with our environment now.
* You may find the demo code on using the agent [here](https://github.com/oliverwang15/Stock-Trading-Gym-Environment/blob/main/demo.ipynb).

zhumingpassional commented:
thanks for your effort.

if the new code is better, can we replace the old version?

oliverwang15 commented:
Hi! I have replaced the old version with the new one. In case this action make conflicts, I kept the old version below and renamed it with `DRLAgent_old`. We may delete it if no one raises any issues after several months. Please check it out again. Thank you!

zhumingpassional commented:
ok
